### ISSUE TITLE
Undefined method value for Hash

### ISSUE THREAD

**Issue by Laykou** [restaurant-cheetah/sidekiq_unique_digests_cleaner#3]
Could there be a change in Sidekiq? This gem now fails to run:

```
undefined method `value' for #<Hash:0x00007fd21448a920>
Did you mean?  value?
               values
/.rvm/gems/ruby-2.7.1/gems/sidekiq_unique_digests_cleaner-1.0.1/lib/sidekiq_unique_digests_cleaner.rb:22:in `block in stuck_digests'
/.rvm/gems/ruby-2.7.1/gems/sidekiq-6.0.7/lib/sidekiq/api.rb:933:in `block (3 levels) in each'
/.rvm/gems/ruby-2.7.1/gems/sidekiq-6.0.7/lib/sidekiq/api.rb:928:in `each_pair'
/.rvm/gems/ruby-2.7.1/gems/sidekiq-6.0.7/lib/sidekiq/api.rb:928:in `block (2 levels) in each'
```

**Comment by mhoncharov** [comment]
Yeah. It's possible. I'll take a look asap. Thanks

**Comment by EvNomad** [comment]
@mxhoncharov This is a confirmed bug.
@Laykou We already have a proposed fix for that. Thank you!

**Comment by Laykou** [comment]
This is the fix for the reference: https://github.com/restaurant-cheetah/sidekiq_unique_digests_cleaner/pull/4

**Comment by EvNomad** [comment]
Merged #4
